python/lsst/obs/hsc/hscMapper.py

```diff
@@ -225,6 +225,11 @@ def map_linearizer(self, dataId, write=False):
             mapper=self,
             storage=self.rootStorage)
 
+    def map_crosstalk(self, dataId, write=False):
+        """Construct this on-the-fly in Gen2.
```

Comment:
Construct ... what on the fly? A bit more explanation here, please.

Reply:
Updated docstring to be clearer:

Fake the mapping for crosstalk.
Crosstalk is constructed from config parameters, but we need Gen2 butlers to be able to respond to requests for it.  Returning None provides a response that can be used with the config parameters to generate the appropriate calibration.